WordPress/src/test/java/org/wordpress/android/util/experiments/ExPlatTest.kt

```diff
@@ -193,25 +196,71 @@ class ExPlatTest : BaseUnitTest() {
     }
 
     @Test
-    fun `refreshIfNeeded does not interact with store if the user is not authorised`() = test {
+    fun `refreshIfNeeded does not interact with store if the user is not authorised and there is no anonymous id`() = test {
         setupExperiments(setOf(dummyExperiment))
         whenever(accountStore.hasAccessToken()).thenReturn(false)
+        whenever(analyticsTracker.getAnonID()).thenReturn(null)
 
-        exPlat.forceRefresh()
```

Comment:
Note that this a correction of a [typing mistake from the previous PR](https://github.com/wordpress-mobile/WordPress-Android/pull/16363/files#diff-d72d5e9f145cb6ac1ad3423c0c1c1f0a937117fc98cfac95081612962ffefedcR200)